script.js

```diff
@@ -1,12 +1,32 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let images = [];
+let mySound;
+
+function preload() {
+  soundFormats("mp3", "ogg");
+  mySound = loadSound("StarsAndStripesForever.mp3");
+  img1 = loadImage("fireworks1.png");
+  img2 = loadImage("fireworks2.png");
+  img3 = loadImage("fireworks3.png");
+  img4 = loadImage("fireworks4.png");
+  images = [img1, img2, img3, img4];
+  
+}
 
 function setup() {
   createCanvas(windowWidth, windowHeight);
-    background(255);
+  background("#383838");
+}
+
+function mouseClicked() {
+  mySound.stop();
+  mySound.play();
 }
 
 function draw() {
   noStroke();
-  fill(random(colorlist));
-  ellipse(mouseX, mouseY, 25, 25);
+  if (frameCount % 20 == 0) {
+    let currImage = random(images);
+    let size = random(50, 200);
+    image(currImage, mouseX, mouseY, size, size);
```

Comment:
one last idea I had is that it could be nice for the fireworks to come from the middle of the mouse rather than the corner. can you think of how you might do that?